packages/sdk/package.json

```diff
@@ -23,20 +23,20 @@
     "check-format": "prettier --list-different --config ./.prettierrc.json --ignore-path ./.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-* temp types *.tgz *.log",
     "format": "prettier --write --config ./.prettierrc.json --ignore-path ./.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser-case": "karma start --single-run --glob=./dist-test/index.integration.browser.js",
-    "integration-test:browser": "node test/integration/scripts/DownloadSimpleAuth.js && concurrently --kill-others --success first \"node test/integration/scripts/StartSimpleAuth.js\" \"npm run integration-test:browser-case\"",
-    "integration-test:node": "mocha -r test/mocha.env.ts -r ts-node/register \"./test/integration/node/*.spec.ts\" --timeout 100000 --exit --reporter ./tools/mocha-multi-reporter.js",
-    "test:integration": "npm run integration-test:node && npm run integration-test:browser",
+    "e2e-test:browser-case": "karma start --single-run --glob=./dist-test/index.e2e.browser.js",
+    "e2e-test:browser": "node test/e2e/scripts/DownloadSimpleAuth.js && concurrently --kill-others --success first \"node test/e2e/scripts/StartSimpleAuth.js\" \"npm run e2e-test:browser-case\"",
+    "e2e-test:node": "mocha -r test/mocha.env.ts -r ts-node/register \"./test/e2e/node/*.spec.ts\" --timeout 100000 --exit --reporter ./tools/mocha-multi-reporter.js",
+    "test:e2e": "npm run e2e-test:node && npm run e2e-test:browser",
     "lint:staged": "lint-staged",
     "lint": "eslint \"**/*.ts\"",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run test:unit && npm run test:integration",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run e2e-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run e2e-test:node",
+    "test": "npm run test:unit && npm run test:e2e",
     "unit-test:browser": "karma start --single-run --glob=./dist-test/index.unit.browser.js",
```

Comment:
can we have coverage report for browser libs?

Reply:
I'll investigate how to add coverage for browser tests later.